src/app/util/ember-compatibility-functions.cpp

```diff
@@ -65,15 +65,17 @@ bool IMEmberAfSendDefaultResponseWithCallback(EmberAfStatus status)
         return false;
     }
 
-    chip::app::CommandPathParams returnStatusParam = { imCompatibilityEmberApsFrame.sourceEndpoint,
+    chip::app::CommandPathParams returnStatusParam = { imCompatibilityEmberApsFrame.destinationEndpoint,
```

Comment:
This change doesn't look right...  Why is it being made?

Reply:
Just for this change: the `imCompatibilityEmberApsFrame` refers to the ApsFrame struct we built for ember command handlers.
In zigbee, the commands have "Source Endpoint", and in CHIP we no longer have it (see above, the source endpoint is fixed to 1). So the `destinationEndpoint` here refers to the endpoint on the server, which matches the spec -- the endpoint where this status comes from.